api/models/trial.js

```diff
@@ -106,6 +106,9 @@ const Trial = BaseModel.extend({
   records: function () {
     return this.hasMany('Record');
   },
+  risk_of_biases: function () {
```

Comment:
Rename this to `risks_of_bias`. I know I argued for `risk_of_biases` myself, but that's only for our internal structure. The name of this method will appear to our users, so we should use the correct English.